wgpu-hal/src/vulkan/adapter.rs

```diff
@@ -295,6 +300,19 @@ impl PhysicalDeviceFeatures {
             } else {
                 None
             },
+            zero_initialize_workgroup_memory: if effective_api_version >= vk::API_VERSION_1_3
+                || enabled_extensions.contains(&vk::KhrZeroInitializeWorkgroupMemoryFn::name())
+            {
+                Some(
+                    vk::PhysicalDeviceZeroInitializeWorkgroupMemoryFeatures::builder()
+                        .shader_zero_initialize_workgroup_memory(
+                            private_caps.zero_initialize_workgroup_memory,
+                        )
+                        .build(),
+                )
+            } else {
+                None
```

Comment:
question: would a diagnostic via `tracing` be interesting to acknowledge that zero-initialization was requested but unsupported?

Reply:
Doesn't seem we do this for the other extensions. Not sure how useful it is.